.github/workflows/merge.yaml

```diff
@@ -0,0 +1,98 @@
+          deleted:
+            - deleted: [ '*.csl', '*.xml' ]
+
+    - name: Changed files
+      if: github.event_name == 'push'
+      run: |
+        echo updated: ${{ steps.release.outputs.updated_files }}
+        echo deleted: ${{ steps.release.outputs.deleted_files }}
+
+    - name: Set up Ruby
+      if: github.event_name == 'workflow_dispatch' || (github.event_name == 'push' && (steps.release.outputs.updated == 'true' || steps.release.outputs.deleted == 'true'))
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: 2.7.1
+    - name: but use cache to speed that up
+      if: github.event_name == 'workflow_dispatch' || (github.event_name == 'push' && (steps.release.outputs.updated == 'true' || steps.release.outputs.deleted == 'true'))
+      uses: actions/cache@v2
+      with:
+        path: vendor/bundle
+        key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-gems-
+    - name: Bundle install
+      if: github.event_name == 'workflow_dispatch' || (github.event_name == 'push' && (steps.release.outputs.updated == 'true' || steps.release.outputs.deleted == 'true'))
+      run: |
+        bundle config path vendor/bundle
+        bundle update sheldon --jobs 4 --retry 3
+
+    - name: Populate new branch
+      run: bundle exec sheldon --token=$GITHUB_TOKEN --verbose --populate release
+      if: github.event_name == 'workflow_dispatch'
+
+    - name: update the timestamps and add the changes
+      run: bundle exec sheldon --token=$GITHUB_TOKEN --verbose --release release ${{ steps.release.outputs.updated_files }}
+      if: github.event_name == 'push' && steps.release.outputs.updated == 'true'
+
+    - name: delete deleted files
+      run: cd release && git rm ${{ steps.release.outputs.deleted_files }}
+      if: github.event_name == 'push' && steps.release.outputs.deleted == 'true'
+
+    - uses: stefanzweifel/git-auto-commit-action@v4
+      with:
+        repository: 'release'
+        commit_message: Releasing ${{ steps.release.outputs.updated_files }} ${{ steps.release.outputs.deleted_files }}
+      if: github.event_name == 'push' && (steps.release.outputs.updated == 'true' || steps.release.outputs.deleted == 'true')
+
+    - uses: stefanzweifel/git-auto-commit-action@v4
+      with:
+        repository: 'release'
+        commit_message: ${{ github.event.inputs.commit_message }}
+      if: github.event_name == 'workflow_dispatch'
+
+    - name: ping distribution-updater
+      if: github.repository == 'citation-style-language/styles'
+      run: |
+        curl -H "Authorization: $DISTRIBUTION_UPDATER_TOKEN" -F 'payload={"type":"pull_request","branch":"master","status":0,"commit":"'$GITHUB_SHA'"}' https://styles-update.zotero.org:8826/
+        curl -H "Authorization: $DISTRIBUTION_UPDATER_TOKEN" -F 'payload={"type":"pull_request","branch":"master","status":0,"commit":"'$GITHUB_SHA'"}' https://styles-update.zotero.org:8827/
```

Comment:
Shouldn't type be something other than `pull_request` here? These will just be skipped on our end.

Reply:
Fixed. I glanced over the fact that it said `!=` rather than `==`.